munit-diff/shared/src/main/scala/munit/diff/Diffs.scala

```diff
@@ -1,5 +1,6 @@
 package munit.diff
 
+@deprecated("Please use Diff instead", "1.0.4")
```

Comment:
```suggestion
@deprecated("Please use Diff instead", "1.1.0")
```
I would say if we are adding anything to API we need a new minor

Reply:
happy to change, but they way i have been interpreting this (and using in scalameta) is using the last version where this functionality was NOT deprecated (since that's basically the current version, which is known), instead of the first version where it will be deprecated (which is the next version, and yet to be decided).

so, 1.0.4 is the last released one. please confirm you'd like me to change to what we think the next one will be.

Reply:
Ach, right. I thought we released 1.0.3 last. my bad. This is fine.